Jenkinsfile

```diff
@@ -77,7 +86,7 @@ pipeline {
       }
       steps {
         sh '''
-          docker build --no-cache --pull --build-arg NGINX_IMAGE_TAG="${BASE_NGINX_IMAGE_TAG}" --build-arg HUGO_VERSION="${hugo_version}" --build-arg NODE_VERSION="${node_version}" -t ${IMAGE_NAME}:${TAG_NAME} -t ${IMAGE_NAME}:latest .
+          docker build --pull --build-arg NGINX_IMAGE_TAG="${BASE_NGINX_IMAGE_TAG}" --build-arg HUGO_VERSION="${hugo_version}" --build-arg NODE_VERSION="${node_version}" -t ${IMAGE_NAME}:${TAG_NAME} -t ${IMAGE_NAME}:latest .
```

Comment:
The `--no-cache` is useless. Because of the way docker cache works, as soon as there are changes in the hugo sources (e.g. when packages-lock.json is modified), the lines after 

COPY . /workdir/

will get re-executed (ie npm ci etc...), but everything before will be using the cache (except when HUGO_VERSION or NODE_VERSION changed). So, in our case, `--no-cache` only provides slower builds :)